src/encoded/static/components/matrix_brain.js

```diff
@@ -336,7 +336,7 @@ const getDataTableData = (context) => {
         });
 
         const uniqueDiseases = Array.from(availableDiseases);
-        const genderSymbol = genderData === 'male' ? '♂' : '♀';
+        const genderSymbol = genderData === 'male' ? 'mars' : 'venus';
```

Comment:
"mars" and "venus" may not make sense to people who are familiar with Roman history

Reply:
I originally called them “male” and “female” until I noticed fontawesome called the corresponding icons “mars” and “venus.” I figured they might have done some research on what to call them (though they certainly might not have).